ckanext/restricted/tests/test_performance_package_search.py

```diff
@@ -0,0 +1,117 @@
+# encoding: utf-8
+import time
+import os.path
+
+from ckan.tests import helpers
+import logging
+import pytest
+from assertpy import assert_that
+from ckan.common import config
+import subprocess
+
+import ckanext.restricted.action
+import ckanext.restricted.plugin
+import ckanext.restricted.tests.old_action
+
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def import_performance_data(clean_db, clean_index):
+    raw_db_url = config['sqlalchemy.url']
+    ckan_dir = get_ckan_directory()
+    sql_file = f'{ckan_dir}/ckanext-restricted/ckanext/restricted/tests/performance_test_data.sql'
+    cmd = ['psql',  f'{raw_db_url}', "-f", f'{sql_file}', '> /dev/null']
+
+    log.info(f"Loading performance data using: '{cmd}'")
+    subprocess.run(cmd)
+    ini_file = f"{ckan_dir}/ckanext-restricted/test.ini"
+
+    cmd = [get_ckan_binary_path(), '-c', ini_file, 'search-index', 'rebuild']
+    log.info(f"Rebuild indexes: {cmd}")
+    subprocess.run(cmd)
+
+
+def get_ckan_directory():
+    candidates = ['/usr/lib/ckan/submodules', '/__w/ckanext-restricted']
```

Comment:
This will only work in some very specific setups. Right now probably only for our local dev env docker setup and github actions runtime.

It is possible to get the absolute directory of the python file. Could this be refactored into a relative directory to the absolute path of the module? `FileNotFoundError` could be extended a little bit to explain that ckan sources are expected at "..." relative path from ckanext-restricted.

Reply:
Yep, good idea.